Include/triliteration.h

```diff
@@ -140,15 +145,17 @@ std::pair<float32_t, uint8_t> estimate_latitude(
  * 
  * @note See the start of hydrophones.hpp for more information
  * 
- * @param time_port Time the signal was measured on the port side
+ * @param lag_port Time the signal was measured on the port side
  * 
- * @param time_starboard Time the signal was measured on the 
+ * @param lag_starboard Time the signal was measured on the 
  * starboard side  
  * 
- * @param time_stern Time the signal was measured at the stern 
+ * @param lag_stern Time the signal was measured at the stern 
  */
-uint8_t estimate_longitude(uint32_t time_port, uint32_t time_starboard,
-            uint32_t time_stern);
+uint8_t estimate_longitude(
+            uint32_t lag_port, 
```

Comment:
Does it make sense to use the array of lags as well here? I don't have a clear idea of how these functions are used, so I can't say for sure.

Reply:
Not a problem in doing it, however I'm contemplating changing the entire estimation towards another way of trilateration. Changing the code here could result in some unecessary work

Reply:
Fair enough, it is not very hard to change down the line either, just something to keep in mind 😄